assignments/prototypes.js

```diff
@@ -10,38 +10,74 @@
   
 /*
   === GameObject ===
-  * createdAt
-  * name
-  * dimensions (These represent the character's size in the video game)
-  * destroy() // prototype method that returns: `${this.name} was removed from the game.`
+  *11 createdAt
+  * 11name
+  * 11dimensions (These represent the character's size in the video game)
+  * 11destroy() // prototype method that returns: `${this.name} was removed from the game.`
+  * 
+  * DONE
 */
+"use strict"
+function GameObject (attributes) {
+  this.createdAt = attributes.createdAt;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+}
+
+GameObject.prototype.destroy = function  () {
+  return (`${this.name} was removed from the game`)
+}
 
 /*
   === CharacterStats ===
-  * healthPoints
-  * takeDamage() // prototype method -> returns the string '<object name> took damage.'
-  * should inherit destroy() from GameObject's prototype
+  11* healthPoints
+  11* takeDamage() // prototype method -> returns the string '<object name> took damage.'
+  11* should inherit destroy() from GameObject's prototype
 */
 
+function CharacterStats (attribute) {
+  GameObject.call(this, attribute)
+  this.healthPoints = attribute.healthPoints;
+}
```

Comment:
Great job binding the `this` keyword as the first thing, and passing that `call` your new attributes attached to the child component here, giving it access to the parents attributes. (almost like linking this template, to the parent template as an attachment.